rllib/examples/policy/random_policy.py

```diff
@@ -30,6 +30,14 @@ def __init__(self, *args, **kwargs):
         else:
             self.action_space_for_sampling = self.action_space
 
+        # Disable view requirements for all columns except OBS, which is
```

Comment:
**question**: why is this needed? seems a bit ad-hoc to exclude obs and next_obs.

Reply:
notice obs column is actually needed by this policy below.
this is a custom implemented policy, and doesn't inherit DynamicTF or EagerTF or Torch policies.
so it doesn't have the "automatically figuring out which columns are needed for training and for action computation" functionality.
the VR specs then need to be manually specified, otherwise Connector will try to stack infos and other columns.

Reply:
Then why don't you override the `init_viewrequirements()` function and do this there?
```
def init_view_requirements(self):
    super().init_view_requirements()
    # Disable view requirements for all columns except OBS, which is
    # actually used.
    for k, v in self.view_requirements.items():
        if k == SampleBatch.OBS or k == SampleBatch.NEXT_OBS:
            continue
        v.used_for_training = False
        v.used_for_compute_actions = False
```

Reply:
oh yeah ... this is exactly why we have code reviews :)
thanks